packages/core/src/data-module/data-cache/dataReducer.ts

```diff
@@ -67,7 +70,8 @@ export const dataReducer: Reducer<DataStreamsStore, AsyncActions> = (
           : requestCache,
         id,
         isLoading,
-        isRefreshing: !!streamStore?.isRefreshing,
+        isRefreshing: true,
+        numOutgoingRequests: noOfOutGoingrequest,
```

Comment:
can we directly use "++outGoingrequest" here? why need extra constant.

Reply:
resolved